neo/Network/RPC/RpcServer.cs

```diff
@@ -511,6 +549,50 @@ private JObject Process(string method, JArray _params)
                             return context.ToJson();
                         }
                     }
+                case "sendmanyfrom":
```

Comment:
I'm not sure this really belongs or is needed in the core RPC methods. It is easy enough to formulate the transaction outside and just call `sendrawtransaction`.